app/src/main/java/com/github/epfl/meili/map/MapActivity.kt

```diff
@@ -81,15 +81,23 @@ class MapActivity : AppCompatActivity(), OnMapReadyCallback {
         } else {
             getLocationPermission()
         }
+
+        //map.setOnPoiClickListener(this)
     }
 
+    /*override fun onPoiClick(poi: PointOfInterest) {
```

Comment:
Should I uncomment this method when I rebase to make clicking the poi go to the poi chat log?

Reply:
Yes, for now this is commented because I haven't figured out yet how to test it and I want to be able to merge